src/NServiceBus.Testing/DependencyInjection/ServiceCollection.cs

```diff
@@ -1,26 +1,25 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-/*
- * This file has been modified from its original form in the following ways:
- * - The namespace has been changed
- * - The access modifier has been changed
- * - The license file name in the copyright notice has been changed to match the license location in this repository
- */
-
-using System.Collections;
-using System.Collections.Generic;
-
 namespace MicrosoftExtensionsDependencyInjection
 {
+    /*
```

Comment:
I thought handling 3rd party copied over code is out of scope and we'll do it as `.g.cs` along with a point to the squad about how to handle those exceptions moving forward.